### PR TITLE
[Improvement][server] Refactor code for getTaskLogPath to reduce NPE log during UT

### PR DESCRIPTION
**Pull request by hsupu**
## What is the purpose of the pull request

- Refactor code for getTaskLogPath to reduce NPE log during UT

## Brief change log

- Add `org.apache.dolphinscheduler.server.utils.LogUtils`
- Remove `MasterBaseTaskExecThread.getTaskLogPath`
- Remove `org.apache.dolphinscheduler.server.worker.processor.TaskExecuteProcessor.getTaskLogPath`

## Verify this pull request

This change added tests and can be verified as follows:

- Added `org.apache.dolphinscheduler.server.utils.LogUtilsTest`.
- Tested locally.

### CODE REVIEW

**Comment by CalvinKirs** [comment]
There is a code smell, can you try to solve it?

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/utils/LogUtils.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+import org.slf4j.LoggerFactory;
+
+import ch.qos.logback.classic.sift.SiftingAppender;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.spi.AppenderAttachable;
+
+public class LogUtils {
+
+    private LogUtils() {
```
Comment (yangyichao-mango) [inline dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/utils/LogUtils.java:36]: Please Add throw new unsupportedException.

File: dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/utils/LogUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.sift.SiftingAppender;
+
+@RunWith(MockitoJUnitRunner.class)
+public class LogUtilsTest {
+
+    @Test
+    public void testGetTaskLogPath() {
+        Assert.assertEquals("/", Constants.SINGLE_SLASH);
```
Comment (yangyichao-mango) [inline dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/utils/LogUtilsTest.java:42]: Is this constant used in logutils? If not, we can remove it,

**Review by yangyichao-mango — CHANGES_REQUESTED** [review]

**Review by qiaozhanwei — APPROVED** [review]
+1

**Pull request merged**